controllers/users.js

```diff
@@ -0,0 +1,50 @@
+const { conrollerWraper } = require("../helpers/controllerWraper");
+const {
+  registerUserService,
+  loginUserService,
+  logoutUserService,
+  subscriptionUserService,
+} = require("../secrive/usersServices");
+
+const registrationUser = async (req, res) => {
+  const subscription = await registerUserService(req.body);
+
+  res.status(201).send({ user: { email: req.body.email, subscription } });
+};
+
+const loginUser = async (req, res) => {
+  const { token, subscription } = await loginUserService(req.body);
+
+  res
+    .status(200)
+    .send({ user: { token, email: req.body.email, subscription } });
```

Comment:
```suggestion
    .json({ user: { token, email: req.body.email, subscription } });
```
json для відправки об'єкту

Reply:
всюди виправіть